opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -55,26 +52,88 @@
 
 _EXPORTER_OTLP = "otlp"
 _EXPORTER_OTLP_PROTO_GRPC = "otlp_proto_grpc"
+_EXPORTER_OTLP_PROTO_HTTP = "otlp_proto_http"
 
 _RANDOM_ID_GENERATOR = "random"
 _DEFAULT_ID_GENERATOR = _RANDOM_ID_GENERATOR
 
+_logger = logging.getLogger(__name__)
+
 
 def _get_id_generator() -> str:
     return environ.get(OTEL_PYTHON_ID_GENERATOR, _DEFAULT_ID_GENERATOR)
 
 
-def _get_exporter_names(names: str) -> Sequence[str]:
-    exporters = set()
+def _get_otlp_exporter_entry_point(
+    otlp_protocol: Optional[str],
```

Comment:
not sure has to be `Optional[str]` and not just `str` as you check before you call this function. Alternatively, you can skip the check

Reply:
(also note that if it's `None` a `RuntimeError` will be raised)